cmake/ConfigFileSetting.cmake

```diff
@@ -120,24 +120,19 @@ set(SUNDIALS_USE_GENERIC_MATH 1)
 nrn_check_include_files(dlfcn.h HAVE_DLFCN_H)
 nrn_check_include_files(execinfo.h HAVE_EXECINFO_H)
 nrn_check_include_files(malloc.h HAVE_MALLOC_H)
-nrn_check_include_files(strings.h HAVE_STRINGS_H)
 nrn_check_include_files(sys/types.h HAVE_SYS_TYPES_H)
 nrn_check_include_files(unistd.h HAVE_UNISTD_H)
 
 # =============================================================================
 # Check for standard headers
 # =============================================================================
-check_include_files("dlfcn.h;stdint.h;stddef.h;inttypes.h;stdlib.h;strings.h;string.h;float.h"
-                    STDC_HEADERS)
+check_include_files("dlfcn.h;stdint.h;stddef.h;inttypes.h;stdlib.h;string.h;float.h" STDC_HEADERS)
```

Comment:
Does one still need to look for `string.h`? Why `string.h` and not `cstring`?

Reply:
same for `stdint`, `stddef`, `inttyped`.

One of the problem being that we are never sure (it is not mandatory) than the `XX` version exists in the same time than the `std::XX` version.

Reply:
Could we perhaps remove the whole line? All except `dlfcn.h` are either in the C or C++ standard, and we check for that one just a few lines above with `nrn_check_include_files(dlfcn.h HAVE_DLFCN_H)`.

Reply:
should try yes;)

Reply:
I quickly checked, you removed the last usage